.github/workflows/test.yml

```diff
@@ -20,13 +20,19 @@ jobs:
         include:
           - ckan-version: "2.10"
             solr-image: "2.10-spatial"
+            harvest-tag: 'v1.4.2'
```

Comment:
Do we need different harvest versions? AFAICT all these should support the same py /ckan combinations

Reply:
I'd install the latest harvest version (master) until we need to pin it for some reason

Reply:
removed here https://github.com/ckan/ckanext-spatial/pull/308/commits/1fa6bb6a8a482e01b8e9a11a3a1355aa90c20e78